packages/editor/src/components/block-switcher/style.scss

```diff
@@ -3,6 +3,19 @@
 	height: $icon-button-size;
 }
 
+.components-icon-button.editor-block-switcher__no-switcher-icon {
+	width: $icon-button-size + 6px + 6px;
```

Comment:
Should we consolidate it with `components-icon-button.editor-block-switcher__toggle`? They should be similar in general. The active icon doesn't have a fixed width but share `margin`, `display`, `height` and `padding`.

Reply:
This duplication was removed.

Reply:
Thaks 👍 Looks great!